docs/developer/first-steps/upgrading.md

```diff
@@ -0,0 +1,143 @@
+:::
+
+---
+
+## Upgrading from Roadiz v2.1 to v2.2
+
+For full details, check the [Changelog](https://github.com/roadiz/core-bundle-dev-app/blob/main/CHANGELOG.md#v220-2023-12-12).
+
+### Key changes:
+
+- **Doctrine migrations** are now the default method to upgrade all node-type-related entities.  
+  Run the following command after updating your Roadiz dependencies:
+
+  ```bash
+  bin/console doctrine:migrations:migrate;
+  ```
+
+- **Roadiz updated to API Platform new version and Metadata scheme.**  
+  You must rewrite your API resource YAML files to match the new scheme.  
+  See the [API Platform upgrade guide](https://api-platform.com/docs/core/upgrade-guide/).
+
+  To regenerate API resources (this will remove any custom serialization groups):
+
+  ```bash
+  bin/console generate:api-resources;
+  ```
+
+- **Node-type updates after Roadiz 2.2 will be versioned** and **will generate a Doctrine migration file.**  
+  If needed, generate a migration file for existing node-types:
+
+  ```bash
+  bin/console doctrine:migrations:generate;
+  ```
+
+- **Entities path change for `roadiz/models`**:  
+  From
+  ```
+  %kernel.project_dir%/vendor/roadiz/models/src/Roadiz/Core/AbstractEntities
+  ```
+  To
+  ```
+  %kernel.project_dir%/lib/Models/src/Core/AbstractEntities
+  ```
+
+- **`Logger` now has a separate entity manager** to avoid persisting invalid log entries.
+
+### Example configuration:
+
+```yaml
+orm:
+    auto_generate_proxy_classes: true
+    default_entity_manager: default
+    entity_managers:
+        # Put `logger` entity manager first to select it as default for Log entity
+        logger:
+            naming_strategy: doctrine.orm.naming_strategy.underscore_number_aware
+            mappings:
+                RoadizCoreLogger:
+                    is_bundle: false
+                    type: attribute
+                    dir: '%kernel.project_dir%/vendor/roadiz/core-bundle/src/Logger/Entity'
+                    prefix: 'RZ\Roadiz\CoreBundle\Logger\Entity'
+                    alias: RoadizCoreLogger
+        default:
+            dql:
+                string_functions:
+                    JSON_CONTAINS: Scienta\DoctrineJsonFunctions\Query\AST\Functions\Mysql\JsonContains
+            naming_strategy: doctrine.orm.naming_strategy.underscore_number_aware
+            auto_mapping: true
+            mappings:
+                RoadizCoreLogger:
+                    is_bundle: false
+                    type: attribute
+                    dir: '%kernel.project_dir%/vendor/roadiz/core-bundle/src/Logger/Entity'
+                    prefix: 'RZ\Roadiz\CoreBundle\Logger\Entity'
+                    alias: RoadizCoreLogger
+                App:
+                    is_bundle: false
+                    type: attribute
+                    dir: '%kernel.project_dir%/src/Entity'
+                    prefix: 'App\Entity'
+                    alias: App
+```
```

Comment:
Maybe move this part to `UPGRADE.md` file a project root. And make a link from documentation